lizzy/api.py

```diff
@@ -89,18 +96,19 @@ def create_stack(new_stack: dict) -> dict:
                    'parameters': parameters}
     tags = {'LizzyKeepStacks': keep_stacks,
             'LizzyTargetTraffic': new_traffic}
-    if senza.create(senza_yaml, stack_version, image_version, parameters,
-                    disable_rollback, tags):
+    try:
+        senza.create(senza_yaml, stack_version, image_version, parameters,
+                     disable_rollback, tags)
+    except ExecutionError as error:
```

Comment:
In this case if an execution error happens it's mostly likely caused by a bad request not a server error.